siliconcompiler/tools/openroad/sc_constraints.sdc

```diff
@@ -14,8 +15,47 @@ if {[dict exists $sc_cfg datasheet] && [dict exists $sc_cfg datasheet $sc_design
             set jittertuple [dict get $sc_cfg datasheet $sc_design pin $pin tjitter global]
             set jitter [sta::time_sta_ui [lindex $jittertuple 1]]
 
-            create_clock [get_ports $pin] -name $pin -period $period
+            utl::info FLW 1 "Creating clock $pin with [sta::format_time [sta::capacitance_ui_sta $period] 3][sta::unit_scale_abreviation time]s period and [sta::format_time [sta::capacitance_ui_sta $jitter] 3][sta::unit_scale_abreviation time]s jitter."
+            create_clock -name $pin -period $period $pin
             set_clock_uncertainty $jitter [get_clock $pin]
         }
     }
 }
+
+### Create IO constraints
+set buffer_cell "NULL"
+foreach cell [get_lib_cells *] {
```

Comment:
Maybe later we could add an OpenROAD schema variable to let people choose this cell, in case they don't want the first available buffer?

Reply:
I've logged it